### ISSUE TITLE
NTR: X-ray: jaw cystic lesions (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8929]
New term request X-ray: jaw cystic lesions (elthat_100830172955): Mandible, mandibular; maxillary  UMLS: 0022361

**Comment by MickeySegal** [comment]
As discussed in https://github.com/obophenotype/human-phenotype-ontology/issues/8928 there is overlap with that term.  Here we used the term for both mandible and maxilla.  We need to decide whether to lump or split.  

https://www.ncbi.nlm.nih.gov/books/NBK1137/ suggests that lumping may be better, but there may be other reasons for splitting.

**Comment by pnrobinson** [comment]
@MickeySegal 
let's go with this for now -- please open new item if both new jaw cyst terms are not sufficient

Adding new term: 
- ID: HP:6000595
- label: Mandibular cyst
- parents: [Abnormal mandible morphology (HP:0000277)](https://hpo.jax.org/app/browse/term/HP:0000277)
- pmid: 
- def: A closed sac that has developed in the lower jaw bone (mandibula) and contains fluid or semi-solid material.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8929
- synonyms:
